drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/resources/config/README.txt

```diff
@@ -0,0 +1,3 @@
+This file is needed to preserve this directory (config/).
```

Comment:
I've seen this in a few places.. why the need/introduction of JNDI? Did I miss something somewhere?